release/changes.md

```diff
@@ -1 +1,5 @@
-- [NEW] Run tests against AGP 8.2.0-alpha16
\ No newline at end of file
+- [NEW] Run tests against AGP 8.2.0-alpha16
+- [NEW] Enabled Predictive Test Selection (PTS) by default.
```

Comment:
I don't think any of these PTS related items should be in the release notes since it doesn't affect consumers in any way.